### ISSUE TITLE
quesition on codes

### ISSUE THREAD

**Issue by wwjwy** [HiLab-git/WSL4MIS#1]
Sorry to interrupt you, I noticed that your weakly supervised and semi-supervised codes seem to differ only in the loss function. Therefore, I think that the semi-supervised model can be modified using a loss function to get a simple weakly supervised model. Can i understand this way? Thanks

**Comment by Luoxd1996** [comment]
Hi,
Compared with fully-supervised learning, most of the semi/weakly-supervised learning just used different training strategies to learn from imperfect annotation. So I think your understanding is right.
Best,
Xiangde.
